### DISCUSSION TITLE
Anyway to have just outlined chat bubble?

### DISCUSSION THREAD

**Discussion by yang-gojuly** [saadeghi/daisyui#3353] [General]
Hello there,

I've been using DaisyUI for my new project, and it looks great. 

One quick question. 

Is there a way to have a plain chat bubble with just outline? I tried to add border like `border-2`, but the border doesn't extend to the chat bubble tail. If there is a workaround, I would be so very much happy and appreciate it. 

<img width="54" alt="Screenshot 2024-12-30 at 4 45 17 PM" src="https://github.com/user-attachments/assets/d4d0d631-91da-4b29-992e-ba41861ace24" />

Thank you!!!

**Comment by saadeghi** [comment]
Tail is a separate SVG, so It won't inherit the border,  
But if the contrast is too much, maybe you can use a light background color instead of a border.  
Example: https://play.tailwindcss.com/BabChuYU0x
